PS E:\django_learning\ContactForm> python manage.py shell
Python 3.12.2 (tags/v3.12.2:6abddd9, Feb  6 2024, 21:26:36) [MSC v.1937 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> from models import *
Traceback (most recent call last):
  File "<console>", line 1, in <module>
ModuleNotFoundError: No module named 'models'
>>> from .models import *
Traceback (most recent call last):
  File "<console>", line 1, in <module>
KeyError: "'__name__' not in globals"
>>> from contact.models import *
>>> User.objects.all()
<QuerySet [<User: User object (1)>, <User: User object (2)>, <User: User object (3)>, <User: User object (4)>]>
>>> User.objects.all()[0].fname
'Anmol R'
>>> User.objects.all()[0].image
<ImageFieldFile: None>
>>> User.objects.all()[1].image 
<ImageFieldFile: img/429909.jpg>
